src/formatDate.js

```diff
@@ -8,7 +8,43 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const partsDate = date.split(fromFormat[fromFormat.length - 1]);
```

Comment:
Consider destructuring 'fromFormat' to explicitly extract the separator, which can improve code readability. For example: 'const [,,,...,separator] = fromFormat;'.